docs/tutorials/blog.md

```diff
@@ -855,35 +855,13 @@ export async function createPost(post: Prisma.PostUncheckedCreateInput) {
 
 ```
 
-```tsx filename=app/routes/posts/admin/new.tsx lines=[1,7]
-import type { ActionArgs } from "@remix-run/node";
```

Comment:
Section not necessary after implementing changes to the tutorial for 1.6.5 release. The types are already inferred.